src/develop/develop.c

```diff
@@ -3187,10 +3185,8 @@ int dt_dev_distort_backtransform_locked(dt_develop_t *dev,
                && module->iop_order <= iop_order)
            || (transf_direction == DT_DEV_TRANSFORM_DIR_BACK_EXCL
                && module->iop_order < iop_order))
-       && !(dev->gui_module
-            && dev->gui_module != module
-            && (dev->gui_module->operation_tags_filter() & module->operation_tags())
-            && (pipe->type & (DT_DEV_PIXELPIPE_FULL | DT_DEV_PIXELPIPE_PREVIEW))))
+       && !(dt_iop_module_is_skipped(dev, module)
+            && (pipe->type & DT_DEV_PIXELPIPE_BASIC)))
```

Comment:
Isn't this a copy/paste mistake? Should be `pipe->type & (DT_DEV_PIXELPIPE_FULL | DT_DEV_PIXELPIPE_PREVIEW`?

Reply:
The binary or is already defined in pixel pipe.h...

Reply:
I see, `BASIC` is `FULL | PREVIEW`, sorry for the noise!